src/perf.cmd.js

```diff
@@ -133,21 +133,21 @@ class PerfCommand {
    * @param {*} limit
    * @returns true if successful, false if unsuccessful and undefined if the name isn't valid
    */
-  static format(metrics, name, limit) {
+  static format(metrics, name, limit, logger = makeLogger()) {
     const metric = metrics.filter(m => m.name === name).length === 1
       ? metrics.filter(m => m.name === name)[0] : null;
     if (metric && metric.name.endsWith('-score')) {
-      console.log(`  ${chalk.gray(`${metric.label}: `)}${PerfCommand.formatScore(metric.value, limit)}`);
+      logger.info(`  ${chalk.gray(`${metric.label}: `)}${PerfCommand.formatScore(metric.value, limit)}`);
       return PerfCommand.formatScore(metric.value, limit).indexOf('(failed)') === -1;
     } if (metric) {
-      console.log(`  ${chalk.gray(`${metric.label}: `)}${PerfCommand.formatMeasure(metric.value, limit)}`);
+      logger.info(`  ${chalk.gray(`${metric.label}: `)}${PerfCommand.formatMeasure(metric.value, limit)}`);
       return PerfCommand.formatMeasure(metric.value, limit).indexOf('(failed)') === -1;
     }
     return undefined;
   }
 
-  static formatResponse(response, params = {}, strainname = 'default') {
-    console.log(`\nTesting ${response.url} on ${response.device.title} (${response.connection.title}) from ${response.location.emoji}  ${response.location.name} using ${strainname} strain.\n`);
+  static formatResponse(response, params = {}, strainname = 'default', logger = makeLogger()) {
```

Comment:
do you need to keep this as static? consider turning into instance method.

Reply:
Keeping it static makes testing easier.